charts/posthog/README.md

```diff
@@ -90,8 +92,13 @@ Dependent charts can also have values overwritten. Preface values with postgresq
 
 All configuration options can be found in [ALL_VALUES.md](ALL_VALUES.md) or in [values.yaml](values.yaml) file.
 
-### Example `values.yaml`
-
+<details>
```

Comment:
Interesting trick! On the PR review screen the triangle to collapse/uncollapse is in the wrong place though

Reply:
hmm - on my preview that was working fine

Reply:
hence the inline styling

Reply:
worth seeing what it ends up looking like live